### ISSUE TITLE
country select on driver license upload

### ISSUE THREAD

**Issue by SimonHausdorf** [onfido/onfido-ios-sdk#48]
for creating an applicant, the api documentation says:
```
country:
optional 
The country where this applicant will be checked. This must be a three-letter ISO code e.g. GBR or USA. If not provided the default is GBR.
```

if a user goes through the process on our website, we ask for the country and set the country when we create the applicant.

On mobile sdk, when we select drivers license, we get asked for the country from onfido. What is this for and do we really have to ask the user twice for his country?

Also we send the country already with creation of the applicant.

**Comment by SergejLogis** [comment]
We have the similar issue in our project as well 🤓We want to present a list of supported KYC documents which is controlled by our server. However, Onfido wants us to specify country, which we do not have on mobile 🤔

**Comment by seewah** [comment]
Hi sorry for the confusion.

- The applicant country in the API (https://documentation.onfido.com/#applicants) is `The country where this applicant will be checked.` The field is more relevant for region-specific database checks such as https://documentation.onfido.com/#identity-report, and is not actually required for `document` or `facial_similarity` checks. Therefore, if you are not using a database check such as `identity` or `watchlist`, you can see ignore that field.
- The country selection in SDK UI is for a different purpose. It is so that the SDK can provide an optimised experience for a particular document type. An example is that we run barcode detection specifically on US driving licence, which is why we ask the end-user for the document country.

In short, they are for different purposes, and we ask the end-user to confirm their document country in order to provide an optimised experience for a particular document type.
